opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -391,42 +289,12 @@ impl opentelemetry_sdk::logs::LogExporter for UserEventsExporter {
     }
 
     #[cfg(feature = "spec_unstable_logs_enabled")]
-    fn event_enabled(&self, level: Severity, _target: &str, name: &str) -> bool {
-        let (found, keyword) = if self.exporter_config.keywords_map.is_empty() {
-            (true, self.exporter_config.default_keyword)
-        } else {
-            // TBD - target is not used as of now for comparison.
-            match self.exporter_config.get_log_keyword(name) {
-                Some(x) => (true, x),
-                _ => (false, 0),
-            }
-        };
-        if !found {
-            return false;
-        }
-        let es = self
-            .provider
-            .find_set(self.get_severity_level(level), keyword);
-        match es {
-            Some(x) => {
-                let enabled = x.enabled();
-                if !enabled {
-                    otel_debug!(
-                        name: "UserEvents.EventNotEnabled",
-                        level = format!("{:?}",level),
-                        keyword = keyword,
-                    );
-                }
-                enabled
-            }
-            _ => {
-                otel_debug!(
-                    name: "UserEvents.EventSetNotFound",
-                    level = format!("{:?}",level),
-                    keyword = keyword,
-                );
-                false
-            }
+    fn event_enabled(&self, level: Severity, _target: &str, _name: &str) -> bool {        
+        let level = self.get_severity_level(level);
+        let event_set_index = level.as_int() as usize - 1;
+        match self.event_sets.get(event_set_index) {
```

Comment:
we can create the vector creation in such a way to align Otel severity index and vec index, so that look up can be slightly faster.